pkg/mountkexec/mountkexec.go

```diff
@@ -39,17 +40,22 @@ func MountISO(isoPath, mountDir string) error {
 }
 
 // KexecISO boots up a new kernel and initramfs
-func KexecISO(opp *webboot.Distro, path string) error {
+func KexecISO(opp *webboot.Distro, dir string) error {
 	var image boot.OSImage
-	if err := multiboot.Probe(path + opp.Kernel); err == nil {
+	kernelPath := opp.Kernel
+	if !filepath.IsAbs(kernelPath) {
+		kernelPath = filepath.Join(dir, opp.Kernel)
```

Comment:
you can change this later but you could consider using kernelPath instead of opp.Kernel. No big deal.

Reply:
Thats true .Let me make the edit.